addons/account/models/chart_template.py

```diff
@@ -99,35 +99,83 @@ class AccountChartTemplate(models.Model):
     account_ids = fields.One2many('account.account.template', 'chart_template_id', string='Associated Account Templates')
     tax_template_ids = fields.One2many('account.tax.template', 'chart_template_id', string='Tax Template List',
         help='List of all the taxes that have to be installed by the wizard')
+
+    # Code prefixes are used to create Suspense Account, Outstanding Payments, Outstanding Receipts accounts
+    # if those are not set implicitly in the chart template.
+    # It could be also used to create `default_account_id` in `account.journal`
     bank_account_code_prefix = fields.Char(string='Prefix of the bank accounts', required=True)
     cash_account_code_prefix = fields.Char(string='Prefix of the main cash accounts', required=True)
+    # Transfers accounts are used for Inter-Banks Transfers.
     transfer_account_code_prefix = fields.Char(string='Prefix of the main transfer accounts', required=True)
+
+    # Currency exchange accounts
     income_currency_exchange_account_id = fields.Many2one('account.account.template',
         string="Gain Exchange Rate Account", domain=[('internal_type', '=', 'other'), ('deprecated', '=', False)])
     expense_currency_exchange_account_id = fields.Many2one('account.account.template',
         string="Loss Exchange Rate Account", domain=[('internal_type', '=', 'other'), ('deprecated', '=', False)])
+
     country_id = fields.Many2one(string="Country", comodel_name='res.country', help="The country this chart of accounts belongs to. None if it's generic.")
 
+    # Payments accounts.
+    # Suspense account is used to register an unreconiled bank entries created via bank statement.
+    # Once it's reconciled, the suspense account in `account.move` will be replaced with a proper account.
+    # Suspense account is an exceptional case when account is supposed to be replaced in the transaction.
+    # The new account depends on `account.payment` records. There are two cases
+    # 1. `account.payment' exists before reconciliation aka Blue lines in reconciliation wizard
+    # 2. `account.payment' is created on reconciliation aka regular Black lines in reconciliation wizard
+    #
+    # For case n.1 the new account is Outstanding Receipts/Payments account and the workflow is following:
+    #
+    # Initial transactions:
```

Comment:
good idea to document but not so sure it's clear... When I read  
    # * Invoice/bill: Payable/Receivable vs Sales/Income
 Then I might be confused because it's not respectively exact.
 Small change but would be more accurate.  
    # * Invoice/bill : Receivable/Payable vs Sales/Income

Reply:
Good point. Fixed